arch/arm/src/stm32h7/stm32_oneshot.c

```diff
@@ -183,14 +183,14 @@ int stm32_oneshot_initialize(struct stm32_oneshot_s *oneshot, int chan,
 {
   uint32_t frequency;
 
-  tmrinfo("chan=%d resolution=%d usec, USEC_PER_SEC:%d\n", chan, resolution,
+  tmrinfo("chan=%d resolution=%d usec, USEC_PER_SEC:%ld\n", chan, resolution,
           USEC_PER_SEC);
   DEBUGASSERT(oneshot && resolution > 0);
 
   /* Get the TC frequency the corresponds to the requested resolution */
 
   frequency = USEC_PER_SEC / (uint32_t)resolution;
-  tmrinfo("frequency: %d\n", frequency);
+  tmrinfo("frequency: %ld\n", frequency);
```

Comment:
```suggestion
  tmrinfo("frequency: %" PRIu32 "\n", frequency);
```

Reply:
I meant to use PRI macros, I think this at some point I messed this up with a prior version of my patch. Changing.